migration/migrate_parameters_from_study.sql

```diff
@@ -0,0 +1,139 @@
+/* How to rollback in case of error?
+ * There is multiples migrations done by this script (se "migrate" variable),
+ *  so only rollback the migration who failed.
+ * If it failed at the end of the script, check before if there wasn't non-rollback-able update/delete!
+ *  If you're not sure, ask GridSuite devs or read this script in detail.
+ * To rollback a single migration who failed up to around half way:
+ *   $ truncate table <to_schema>.<to_table>
+ *   $ update study.study set <from_old_id>=coalesce(<from_old_id>, <from_new_uuid>), <from_new_uuid>=null
+ */
+DO
+$body$
+<<fn>>
+DECLARE
+    study_name text := quote_ident('study');
+    migrate constant jsonb[] := array[
+        '{"from_table": "short_circuit_parameters", "from_old_id": "short_circuit_parameters_entity_id", "from_new_uuid": "short_circuit_parameters_uuid", "to_schema": "shortcircuit", "to_table": "analysis_parameters"}'
+    ];
+    params jsonb;
+    path_from text;
+    path_to text;
+    remote_databases bool;
+    insert_columns text;
+    rows_affected integer;
+    err_returned_sqlstate text;
+    err_column_name text;
+    err_constraint_name text;
+    err_pg_datatype_name text;
+    err_message_text text;
+    err_table_name text;
+    err_schema_name text;
+    err_pg_exception_detail text;
+    err_pg_exception_hint text;
+    err_pg_exception_context text;
+BEGIN
+    --lock table study in exclusive mode;
+    raise log 'Script starting at %', now();
+    raise log 'user=%, db=%, schema=%', current_user, current_database(), current_schema();
+
+    /* Case OPF: copy from db.study.<table> to db.<service>.<table> (easy, no problem to migrate)
+     * Case localdev & Azure: copy from study.public.<table> to <service>.public.<table> (need to copy between databases...)
+     *   [0A000] cross-database references are not implemented: "ref.to.remote.table"
```

Comment:
Don't undersand, cross-database it's not possible ?

Reply:
not without using an extension (a data wrapper) with what they call "foreign table" with an "foreign server"